src/main/java/view/ResultView.java

```diff
@@ -1,37 +1,51 @@
 package view;
 
+import model.CarName;
 import model.Position;
 
+import java.util.List;
+import java.util.stream.Collectors;
+
 public class ResultView {
 
     private static final String PRINT_RESULT = "실행 결과";
     private static final String PRINT_POSITION = "-";
+    private static final String CAR_NAME_POSITION_SEPARATOR = " : ";
+    private static final String CAR_NAME_COLLECTORS_JOINING = ", ";
+    private static final String WINNER_CAR_NAME_APPEND_TEXT = "가 최종 우승했습니다.";
 
-    private ResultView() {
-
-    }
-
-    public static ResultView getInstance() {
-        return new ResultView();
-    }
 
-    public void printResult() {
+    public static void printResult() {
         System.out.println(PRINT_RESULT);
     }
 
-    public void printLineBreak() {
+    public static void printLineBreak() {
         System.out.println();
     }
 
-    public void printPosition(Position position) {
+    public static void printExecuteResult(CarName carName, Position position) {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append(carName.getName());
+        stringBuilder.append(CAR_NAME_POSITION_SEPARATOR);
+        stringBuilder.append(printPosition(position));
```

Comment:
StringBuilder의 append 메서드는 반환타입이 자기자신이기에 메서드체이닝이 가능합니다. 
```suggestion
        stringBuilder.append(carName.getName())
                             .append(CAR_NAME_POSITION_SEPARATOR)
                             .append(printPosition(position));
```